### ISSUE TITLE
Not installable via composer

### ISSUE THREAD

**Issue by pclaisse** [b13/masi#5]
The package is not installable via composer. I always get the following message:
```
Could not find a matching version of package b13/masi. Check the package spelling, your version constraint and that the package is available in a stability which matches your minimum-stability (stable).
```

**Comment by davidsteeb** [comment]
It seems the package was no longer available on packagist.org – I readded the package and hope this solves your problem. If not, please let me know.

**Comment by pclaisse** [comment]
Still the same error

**Comment by bmack** [comment]
@pclaisse this was because we haven't made a release yet. I just published v1.0.0. Feel free to check it out again, as you now have a stable release. https://packagist.org/packages/b13/masi v1.0.0

**Comment by pclaisse** [comment]
Thank you for the quick response! Now it worked ;-)
